content/1_docs/1_guide/1_quickstart/guide.txt

```diff
@@ -179,43 +179,55 @@ If you want to upgrade an existing Kirby 2 installation, head over to our (link:
 
 ## Where to go from here
 
-### A tour around the house
-Now that you have successfully installed Kirby on your server, it's time to explore your new Kirby installation. So get yourself a cup of coffee or tea (or whatever else you like most) and head over to the sections that interest you most or go on a tour to explore the Starterkit with us.
+(columns…)
 
-(link: docs/guide/tour text: Explore the Starterkit ›)
+### (link: docs/guide/tour text: A tour around the house)
+You have successfully installed Kirby on your server, it's time to explore your new CMS. Get yourself a cup of tea (or whatever else you like most) and go on (link: docs/guide/tour text: a tour to explore the Starterkit) with us.
```

Comment:
Das klingt komisch, ich würde die ursprüngliche Satzstruktur beibehalten, `Now that you...`